driver-core/src/main/com/mongodb/client/model/expressions/MapExpression.java

```diff
@@ -23,11 +23,11 @@
 
 /**
  * A map {@link Expression value} in the context of the MongoDB Query
- * Language (MQL). An map is a finite, unordered a set of
- * {@link EntryExpression entries} of a certain type, such that no entry key
- * is repeated. It is a mapping from keys to values.
+ * Language (MQL). A map is a finite set of
```

Comment:
"finite" is stated here but is not in the `DocumentExpression`. As a reader of the documentation I am now confused whether this is because a document may be an infinite set of fields?